pkgs/by-name/te/terraform-mcp-server/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "terraform-mcp-server";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = "terraform-mcp-server";
+    rev = "v${version}";
+    hash = "sha256-HYiA0Mfp87czQShiXbS+y20yQzxTn0+hfM0M1kLFZFM=";
+  };
+
+  vendorHash = "sha256-m4J2WGcL0KB1InyciQEmLOSBw779/kagUOIkTwc4CE4=";
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.commit=${src.rev}"
+  ];
+
+  subPackages = [ "cmd/terraform-mcp-server" ];
+
+  postInstall = ''
+    mkdir -p $out/share/${pname}
+
+    cp README.md LICENSE $out/share/${pname}/ || true
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Removing `with lib;` is greatly encouraged.